mesonbuild/coredata.py

```diff
@@ -659,7 +671,7 @@ def get_option(self, key: OptionKey) -> T.Union[T.List[str], str, int, bool, Wra
             v = self.options[key].value
             if key.name == 'wrap_mode':
                 return WrapMode[v]
-            return v
+            return T.cast(T.Union[T.List[str], str, int, bool], v)
```

Comment:
Please quote casts, the first argument to T.cast() is not stringized by `from __future__ import annotations`.